src/styles/blocks/_menu.scss

```
@@ -0,0 +1,93 @@
+.menu {
+  background-color: #c2c2c2;
+  transition: $transition;
+  &__content {
+    padding: $header-top-padding 0;
+    height: 100vh;
+
+    @include onTablet() {
+      //height: 630px;
+      padding-top: $padiing-top-at-header-on-desktop;
+    }
+  }
+
+  @include onDesktop() {
+    //height: 711px;
+  }
+
+  .menu-icons {
+    justify-content: space-between;
+  }
+
+  .menu-img {
+    width: 130px;
+    height: 38px;
+    display: flex;
+
+    @include onDesktop() {
+      width: 187px;
+      height: 56px;
+    }
+  }
+
+  &__top {
+    margin-bottom: 42px;
+    @include top-actions;
+  }
+  
+  &__list {
+    padding: 0 0 40px;
+    margin: 0;
+    list-style: none;
+  }
+
+  &__item {
+    margin-bottom: 24px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    padding-bottom: 8px;
+    border-bottom: 1px solid transparent;
+    color: $basic-color;
+    font-weight: 600;
+    font-size: 22px;
+    line-height: 22px;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    transition: border-color 0.3s;
```

create a variable for $transitionDuration
```suggestion
    transition: border-color $transitionDuration;
```

---

I will fix it, thank you very much for your work!)